src/components/pages/CourseSearch.js

```diff
@@ -0,0 +1,60 @@
+import React, { Fragment, useState, useEffect } from 'react';
+import Requestor from '../Requestor';
+
+const CourseSearch = () => {
+  const [courses, setCourses] = useState([]);
+
+  // Get courses
+  useEffect(() => {
+    getCourses();
+    // eslint-disable-next-line
+  });
+
+  const getCourses = async () => {
+    const coursesResponse = await Requestor.get(
+      '/courses',
+      {},
+      {
+        filters: {},
+        offset: 0,
+        size: 5
+      }
+    );
+    setCourses(coursesResponse.data);
+  };
+
+  const showCourses = courses.map(course => (
+    <tr key={course.id}>
+      <th>{course.id}</th>
+      <th>{course.study_abroad_institution}</th>
+      <th>{course.program_name}</th>
+      <th>{course.year}</th>
+      <th>{course.kzoo_course_name}</th>
+      <th>{course.kzoo_discipline}</th>
+      <th>{course.host_insti_course_number}</th>
+      <th>{course.host_insti_course_name}</th>
+    </tr>
+  ));
+
+  return (
+    <Fragment>
+      <table style={{ width: '100%' }}>
+        <tbody>
+          <tr>
```

Comment:
This entire `tr` should be under `thead`, not as important now because we know we will be refactoring this part of the code when `Table.js` component is ready to be used.